### ISSUE TITLE
Programs run really slow!

### ISSUE THREAD

**Issue by pomeroyk** [JuiceMind/JuiceMind#354]
Some simple java programs are running really slow or just not at all, even though they have no errors.
Hopefully we can expect this to improve over time? 
Thank you!  

-------------------------------------------------
 This week the high school teachers that we work with are still reporting slow execution times in Juicemind even for small programs.

-----------------------------------------------------
Recently when we initially run our programs we get a prompt that lasts around 20 seconds.  I'm not sure if this is the new normal or not.  See screenshot below.

**Comment by JL248** [comment]
Yes, the same is for Python. Even a simple print is not executed immediately.

**Comment by karlr-dev** [comment]
@pomeroyk @JL248 This should be fixed now.
